### DISCUSSION TITLE
Daily challenge doesn't show up on linux

### DISCUSSION THREAD

**Discussion by chicco-carone** [ppy/osu#29130] [Q&A]
### Type

Game behaviour

### Bug description

I updated the game using the package from the aur and went to check for the daily challenge and it isn't present. After asking to one of my friends on windows he has the button and can use it without issues.

### Screenshots or videos

![osu_2024-07-26_22-55-59](https://github.com/user-attachments/assets/26379928-48e8-4071-97c9-de508552e677)

### Version

2024.727.0-lazer

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/16397011/compressed-logs.zip)

**Comment by cadr10** [comment]
I am almost sure AUR packages aren´t officially by osu! dev team. Try grabbing the AppImage from the latest release and running it. 

I am on Arch (CachyOS) and have the Daily Challenge (always running the latest AppImage release).

**Reply by chicco-carone** [reply]
I tried with the appimage and i get the same issue

**Comment by shinobu-uwu** [comment]
> I am almost sure AUR packages aren´t officially by osu! dev team. Try grabbing the AppImage from the latest release and running it.

AUR packages are just bash scripts to install packages, in `osu-lazer-bin`'s case it downloads from the releases page as you can see below, so it must be something different

```bash
source=(
  "$_pkgname-$pkgver.AppImage::https://github.com/ppy/osu/releases/download/$pkgver/osu.AppImage"
  "icon.png"
  "https://raw.githubusercontent.com/ppy/osu-resources/master/LICENCE.md"
  "osu-lazer.desktop"
  "osu-lazer-uri-handler.desktop"
  "osu-lazer"
)
```

**Comment by Stoppedpuma** [comment]
This seems to be an issue on your end as this works fine on my end with the `osu-lazer-bin` AUR package. Your logs have errors in related to network so that might be the cause here?
